tests/connector_test.go

```diff
@@ -124,3 +74,110 @@ func RunStep(t *testing.T, connector deployer.Connector, moduleName string) (str
 
 	return executionResult.OutputID, executionResult.OutputData, executionResult.Error
 }
+
+// This test ensures that this deployer can create and execute
+// connectors concurrently, and that those connectors can deploy
+// plugins concurrently that create side-effects local to their
+// filesystem, and one connector can pull multiple python modules
+func TestDeployConcurrent_ConnectorsAndPluginsWithDifferentModules(t *testing.T) {
+	type TestModule struct {
+		location string
+		stepID   string
+		input    map[string]any
+	}
+	testModules := map[string]TestModule{
+		"fio": {
+			stepID:   "workload",
+			location: "arcaflow-plugin-fio@git+https://github.com/arcalot/arcaflow-plugin-fio.git@de07b3e48cefdaa084eb0445616abc2d13670191",
+			input: map[string]any{
+				"name":    "poisson-rate-submit",
+				"cleanup": "true",
+				"params": map[string]any{
+					"size":           "900KiB",
+					"readwrite":      "randrw",
+					"ioengine":       "sync",
+					"iodepth":        32,
+					"io_submit_mode": "inline",
+					"rate_iops":      50,
+					"rate_process":   "poisson",
+					"buffered":       0,
+				},
+			},
+		},
+		"template": {
+			stepID:   "hello-world",
+			location: "arcaflow-plugin-template-python@git+https://github.com/arcalot/arcaflow-plugin-template-python.git@52d1a9559c60a615dbd97114572f16d70fa30b1b",
+			input: map[string]any{
+				"name": "arca lot",
+			},
+		},
+		"wait": {
+			stepID:   "wait",
+			location: "arcaflow-plugin-wait@git+https://github.com/arcalot/arcaflow-plugin-wait.git",
```

Comment:
If it is important and appropriate for the `location` to include a commit hash, why was it omitted from line 116?  (My instinct is that it _should_ be omitted, but it's probably even more important to be consistent.)

Reply:
`location` was omitted so that the test would explore the `GetModulePath()` branch with `_latest`.